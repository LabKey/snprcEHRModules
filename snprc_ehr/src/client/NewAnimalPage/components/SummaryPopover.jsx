import React, { PureComponent } from 'react';
import {Popover} from 'react-bootstrap';

class SummaryPopover extends PureComponent {

    render() {
        
        return (
            <Popover 
                {...this.props} 
                id="summary-popover"
                className="summary-popover"
            >
                {this.props.message}
            </Popover>
    
        )
    } 
}

export default SummaryPopover;