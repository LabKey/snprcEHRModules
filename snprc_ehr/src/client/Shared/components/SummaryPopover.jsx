import React, { PureComponent } from 'react'
import { Popover } from 'react-bootstrap'

class SummaryPopover extends PureComponent {
    render() {
        return (
          <Popover
            { ...this.props }
            className="summary-popover"
            id="summary-popover"
          >
            {this.props.message}
          </Popover>

        )
    }
}

export default SummaryPopover
