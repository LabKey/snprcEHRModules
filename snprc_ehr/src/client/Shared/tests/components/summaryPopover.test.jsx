import React from 'react'
import { shallow } from 'enzyme'
import SummaryPopover from '../../components/SummaryPopover'

test('Should render the SummaryPopover', () => {
    const wrapper = shallow(
      <SummaryPopover
        message="Summary popover test"
        title="popover"
      />
    )
    expect(wrapper).toMatchSnapshot()
})
