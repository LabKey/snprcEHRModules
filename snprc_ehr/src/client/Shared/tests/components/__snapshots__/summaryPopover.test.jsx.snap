// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the SummaryPopover 1`] = `
<Popover
  bsClass="popover"
  className="summary-popover"
  id="summary-popover"
  message="Summary popover test"
  placement="right"
  title="popover"
>
  Summary popover test
</Popover>
`;
